tests/TestCase/View/Helper/TextHelperTest.php

```diff
@@ -59,7 +64,7 @@ public function setUp(): void
     public function tearDown(): void
     {
         unset($this->Text, $this->View);
-        static::setAppNamespace($this->_appNamespace);
+        static::setAppNamespace($this->appNamespace);
```

Comment:
These shouldn't be necessary since Configure is restored in TestCase::tearDown(). We can remove it separately though.